Monopoly/Assets/Scripts/Game.cs

```diff
@@ -185,20 +183,29 @@ void Update()
             {
                 if (currentPlayerBoughtProperty)
                 {
-                    currentPlayer.Buy(currentPlayerStandingProperty);
-                    currentPlayerStandingProperty.Buy(currentPlayerId);
-                   
+                    moneyManager.WithdrawFromAccount(currentPlayer, currentPlayerStandingProperty.price);
 
+                    if (moneyManager.DoesPlayerHasAnyMoneyLeft(currentPlayer))
+                    {
+                        currentPlayer.Buy(currentPlayerStandingProperty);
+                        currentPlayerStandingProperty.Buy(currentPlayerId);
```

Comment:
Property ma metodę buy? :O trochę to nieintuicyjne, muszą być dwie metody buy, by gracz kupił property? Jedna gracza i druga property?

Reply:
Jest to trochę średnio przemyślane, ale działa
Ja bym w tym już nie grzebał już, bo i tak już nikt z uczynienia tego bardziej czytelnym nie skorzysta

Reply:
Czyli porzucimy nasz projekt? ;(